src/clock/Clock.sol

```diff
@@ -29,6 +29,7 @@ contract Clock is IClock, DaoAuthorizable, UUPSUpgradeable {
                             Initialization
     //////////////////////////////////////////////////////////////*/
 
+    /// @custom:oz-upgrades-unsafe-allow constructor
```

Comment:
This annotation is needed in all the contracts with a constructor (also in the old ones)